accounts/abi/fuzz_utils.go

```diff
@@ -0,0 +1,121 @@
+package abi
```

Comment:
This was implemented in a different file so that we do not have to worry about upstream changes in the future.